hack/collector-default

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# -------------------------------------------------------------------------------- #
+# Description                                                                      #
+# -------------------------------------------------------------------------------- #
+# collctor-default will collect host information as package for support-bundle-kit #
+# e.g., hostname, syslog, dmesg etc                                                #
+# -------------------------------------------------------------------------------- #
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+cd ${BUNDLE_DIR}
+
+# -------------------------------------------------------------------------------- #
+# host information                                                                 #
+# -------------------------------------------------------------------------------- #
+mkdir -p hostinfos
+cd hostinfos
+
+echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+echo "system_info : " `uname -a` >> hostinfo
+
+# Boot Configuration
+cat ${HOST_PATH}/boot/config-$(uname -r) > boot_config
+
+# CPU, Memory and Process usage status
+top -b -n 1 > cpu_memory_process_status
+
+# -------------------------------------------------------------------------------- #
+# collect logs                                                                     #
+# -------------------------------------------------------------------------------- #
+cd ${BUNDLE_DIR}
+mkdir -p logs
+cd logs
+
+dmesg -HTx &> dmesg.log
+
+# syslog
+tail -c 10m ${HOST_PATH}/var/log/syslog > syslog.log
+
+#kernel log
+chroot ${HOST_PATH} /usr/bin/journalctl -k > kernel.log
+
+# kubelet log
+# On Linux nodes that use systemd, the kubelet and container runtime write to journald by default. 
+# ref. https://kubernetes.io/docs/concepts/cluster-administration/logging/#log-location-node
+chroot ${HOST_PATH} /usr/bin/journalctl -r -u kubelet > kubelet.log
+# RKE2
+cp ${HOST_PATH}/var/lib/rancher/rke2/agent/logs/kubelet_rke2.log .
```

Comment:
> cp ${HOST_PATH}/var/lib/rancher/rke2/agent/logs/kubelet_rke2.log .

What is the reason for having an extra copy of the RKE2 kubelet logs?

Reply:
Since the RKE2 kubelet log are save in rke2 agent log folder not journald. So I additionally copy against this log file

Reply:
What happens if the path doesn't exist?

Reply:
I would add check if the file exists to avoid errors.